summarization/extractor_output_preprocessor.py

```diff
@@ -109,7 +118,10 @@ def _split_content(self):
             elif content.content_type == ContentType.IMAGE:
                 self.media_content_list.append(content)
 
-            elif content.content_type != ContentType.UNKNOWN:
+            elif content.content_type == ContentType.QUOTE:
+                self.quoted_content_list.append(content)
+
+            elif content.content_type in self.EMBEDDED_CONTENT_TYPES:
```

Comment:
After adding method in enum, you should simply
`content.content_type.is_embedded_content()`

Reply:
Updated accordingly.